model/Makefile

```diff
@@ -38,15 +38,16 @@ docs/example-with-datasets.md: docs/example-with-datasets.qmd
 	poetry run quarto render docs/example-with-datasets.qmd
 
 docs/py: docs/notebooks
-	jupyter nbconvert --to python docs/pyrenew_demo.ipynb
-	jupyter nbconvert --to python docs/getting-started.ipynb
-	jupyter nbconvert --to python docs/example-with-datasets.ipynb
+	for i in docs/*.ipynb; do \
+		jupyter nbconvert --to python $$i ; \
```

Comment:
No changes suggested here, just a quick question: Have you reviewed the outputted `.py` files? I ask since historically speaking I have had issues with certain notebooks and `nbconvert` properly converting them; might be good to check the output (manually and or programmatically), if this has not already been done.

Reply:
Yes, the outputs are not the best. I use that formula just for debugging. Sometimes, the qmd files err, and it is easier to debug them using python files only.